core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.html

```diff
@@ -44,14 +80,130 @@
       <lazy-loading [hidden]="!opportunityDataIsLoading"></lazy-loading>
     </div>
   </div>
-  <div class="oppia-opportunities-label-container oppia-animate-fadein" *ngIf="labelRequired">
-    <div class="oppia-opportunities-label" [ngStyle]="labelStyle">
-      <span class="protractor-test-opportunity-list-item-label" [hidden]="opportunityDataIsLoading">{{ labelText }}</span>
+  <div class="oppia-opportunities-label-container oppia-animate-fadein"
+       *ngIf="labelRequired">
+    <div class="oppia-opportunities-label"
+         [ngStyle]="labelStyle">
+      <span class="protractor-test-opportunity-list-item-label"
+            [hidden]="opportunityDataIsLoading">
+          {{ labelText }}
+      </span>
+      <lazy-loading [hidden]="!opportunityDataIsLoading"></lazy-loading>
+    </div>
+  </div>
+  <div class="oppia-opportunities-button-container oppia-animate-fadein">
+    <div class="oppia-opportunities-button-wrapper"
+         [disableTooltip]="!opportunityButtonDisabled"
+         ngbTooltip="All available translations are currently in review."
+         placement="bottom">
+      <button class="protractor-test-opportunity-list-item-button"
+              [ngClass]="{'oppia-opportunities-button': !opportunityButtonDisabled, 'oppia-opportunities-disabled-button': opportunityButtonDisabled}"
+              [hidden]="opportunityDataIsLoading || correspondingOpportunityDeleted"
+              [disabled]="opportunityButtonDisabled"
+              (click)="clickActionButton.emit(opportunity.id)">
+        {{ opportunity?.actionButtonTitle }}
+      </button>
+    </div>
+  </div>
+</div>
+
+<!-- Mobile -->
+<div *ngIf="showOpportunityButton && onMobile"
```

Comment:
A lot of the child HTML components seem to be the same and so it will be difficult to make changes in the future—you'd have to check each "version", compare the diff and make changes in each version. Is it possible to extract the common parts into subcomponents?